### DISCUSSION TITLE
Notification after a lockdown is over

### DISCUSSION THREAD

**Discussion by rafa-vasconcelos** [proginosko/LeechBlockNG#355] [Ideas]
I personally use the pomodoro technique and a notification feature would be really great.

Say I want to work focused for an hour, but when I received a notification after that hour I can unwind for a bit (say, 15 minutes) and surf the web and then I can get back to work by initializing another lockdown and so on and so forth.

Thanks in advance and thank you so much for your application, it has worked great for me.

**Comment by proginosko** [comment]
Thanks for the suggestion. Do you mean a notification when the lockdown ends?

**Comment by rafa-vasconcelos** [comment]
Yes! That's it
